app/src/main/java/com/group80/uoftinder/chat/Message.java

```diff
@@ -0,0 +1,50 @@
+package com.group80.uoftinder.chat;
+
+public class Message {
+    String message;
+    String senderId;
+    long timestamp;
+    String currentTime;
```

Comment:
Just a question - what is the necessity behind having both the timestamp and currentTime as instance variables here? My intuition tells me that converting the timestamp into currentTime (string format) and just storing the currentTime would suffice and also be more efficient?

Reply:
I was initially considering using the timestamp to determine which messages are newly created, so that we won't have to clear the message queue and re-add every message once there is an update. This turned out only to create more problems, so I've decided to stick to the inefficient design for now, but leave space for future updates.